spec/services/idv/profile_maker_spec.rb

```diff
@@ -135,28 +136,70 @@
     end
 
     context 'with in_person_verification_needed' do
-      let(:profile) do
-        subject.save_profile(
-          fraud_pending_reason: nil,
-          gpo_verification_needed: false,
-          deactivation_reason: nil,
-          in_person_verification_needed: true,
-          selfie_check_performed: false,
-        )
-      end
-      it 'creates a pending profile for in person verification' do
-        expect(profile.activated_at).to be_nil
-        expect(profile.active).to eq(false)
-        expect(profile.deactivation_reason).to be_nil
-        expect(profile.fraud_pending_reason).to be_nil
-        expect(profile.in_person_verification_pending?).to eq(true)
-        expect(profile.fraud_review_pending?).to eq(false)
-        expect(profile.gpo_verification_pending_at.present?).to eq(false)
-        expect(profile.initiating_service_provider).to eq(nil)
-        expect(profile.verified_at).to be_nil
+      context 'when threatmetrix is disabled' do
+        before do
+          allow(IdentityConfig.store).to receive(:in_person_proofing_enforce_tmx).
+            and_return(in_person_proofing_enforce_tmx_mock)
+        end
+
+        let(:profile) do
+          subject.save_profile(
+            fraud_pending_reason: nil,
+            gpo_verification_needed: false,
+            deactivation_reason: nil,
+            in_person_verification_needed: true,
+            selfie_check_performed: false,
+          )
+        end
+        it 'creates a pending profile for in person verification' do
+          expect(profile.activated_at).to be_nil
+          expect(profile.active).to eq(false)
+          expect(profile.deactivation_reason).to be_nil
+          expect(profile.fraud_pending_reason).to be_nil
+          expect(profile.in_person_verification_pending?).to eq(true)
+          expect(profile.fraud_review_pending?).to eq(false)
+          expect(profile.gpo_verification_pending_at.present?).to eq(false)
+          expect(profile.initiating_service_provider).to eq(nil)
+          expect(profile.verified_at).to be_nil
+        end
+        it 'marks the profile as legacy_in_person' do
```

Comment:
```suggestion

        it 'marks the profile as legacy_in_person' do
```

This feels like an incredibly trivial nit, but these read much better to me with a newline separating cases.